src/api/posts/src/storage.js

```diff
@@ -90,6 +90,20 @@ module.exports = {
     );
   },
 
+  getDelayedFeeds: async () => {
+    const delayedKeys = await getFeedKeysUsingScanStream(`${feedNamespace}*${delayedSuffix}`);
+    return Promise.all(
+      delayedKeys.map(async (key) => {
+        const timeLeft = await redis.ttl(key);
```

Comment:
I can't decide if we need this info or not.  The fact that it will require so many extra `async` calls to Redis makes me want to say that we don't.  Just knowing that it's delayed is probably enough?  What's your thinking?

Reply:
Yeah, I think we should not have this timeLeft infor. we don't use it for anything. I will remove it and fix the test. Thank you :)